source/content/github-actions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Pantheon's GitHub Actions
+description: Actions for web teams and plugin authors
+tags: [continuous-integration, workflow, D8, D9, D10]
+contenttype: [doc]
+innav: [true]
+categories: [automate, workflows]
+cms: [drupal, wordpress]
+audience: [development]
+product: [terminus]
+integration: [--]
+reviewed: "2025-04-15"
+
+---
+
+Pantheon maintains multiple GitHub Actions for integrating with our platform.
+This page focuses on the two actions most commonly used by web teams deploying to Pantheon.
+Separately, [we maintain multiple GitHub Actions meant for authors of WordPress plugins and other packages](https://github.com/pantheon-systems/action-library).
```

Comment:
```suggestion
Pantheon maintains [multiple GitHub Actions](https://github.com/pantheon-systems/action-library) for integrating with our platform or designed for authors of WordPress plugins and other packages.
This page focuses on the two actions most commonly used by web teams deploying to Pantheon.
```

This paragraph repeats itself. Moving the link to the actions library and combining the sentences about "maintain[ing] multiple GitHub Actions" into one.